assignment-section-05/classification_model/pipeline.py

```diff
@@ -0,0 +1,73 @@
+# to persist the model and the scaler
+# for encoding categorical variables
+from feature_engine.encoding import RareLabelEncoder, OneHotEncoder
+
+# for imputation
+from feature_engine.imputation import (
+    CategoricalImputer,
+    AddMissingIndicator,
+    MeanMedianImputer,
+)
+
+# for the preprocessors
```

Comment:
this comment? probably remove

Reply:
removed (and one or two other v. obvious ones)